docs/input-kafka.asciidoc

```diff
@@ -285,6 +285,17 @@ This will add a field named `kafka` to the logstash event containing the followi
 * `offset`: The offset from the partition this message is associated with
 * `key`: A ByteBuffer containing the message key
 
+
+[id="plugins-{type}s-{plugin}-auto_create_topics"]
```

Comment:
Can you add the entry for this option on the list index at the top of the document on line 101?

```
[cols="<,<,<",options="header",]
|=======================================================================
|Setting |Input type|Required
| <<plugins-{type}s-{plugin}-auto_commit_interval_ms>> |<<number,number>>|No
>>> **insert here**
| <<plugins-{type}s-{plugin}-auto_offset_reset>> |<<string,string>>|No
| <<plugins-{type}s-{plugin}-bootstrap_servers>> |<<string,string>>|No
| <<plugins-{type}s-{plugin}-check_crcs>> |<<boolean,boolean>>|No
```